odoo/models.py

```diff
@@ -143,9 +143,9 @@ def raise_on_invalid_object_name(name):
 def check_pg_name(name):
     """ Check whether the given name is a valid PostgreSQL identifier name. """
     if not regex_pg_name.match(name):
-        raise ValidationError("Invalid characters in table name %r" % name)
+        raise ValidationError(_("Invalid characters in table name %(name)s", name=name))
```

Comment:
Same for all errors in this file

Reply:
Just leaving one comment for "technical errors":
Try to use `env._` to translate everywhere. If you cannot find an env, it is probably a technical error and you may not want to translate it.